src/main/java/com/google/jenkins/plugins/k8sengine/client/ContainerClient.java

```diff
@@ -68,13 +69,25 @@ public Cluster getCluster(String projectId, String zone, String cluster) throws
    */
   public List<Cluster> listClusters(String projectId, String zone) throws IOException {
```

Comment:
If the zone parameter is expected to have a null value, it should be wrapped in an Optional.

Reply:
It should never be null because of the way that the zone is provided in the doCheck and doFill methods which call this, but I'd still rather have that behavior not be to throw a NPE. Either way this method will soon actually be replaced with just the listAllClusters method where zone is not a parameter in a follow on pr.